### ISSUE TITLE
toggle-group import's missing file

### ISSUE THREAD

**Issue by coryvirok** [huntabyte/shadcn-svelte#951]
### Describe the bug

After installing toggle-group the new code in $lib/components/ui/toggle-group/index.ts imports from $lib/registry/default/ui/toggle/index.js" instead of $lib/components/ui/toggle/index.ts

https://github.com/huntabyte/shadcn-svelte/blob/main/apps/www/src/lib/registry/default/ui/toggle-group/index.ts#L2

### Reproduction

Install toggle-group

### System Info

```bash
System:
    OS: macOS 13.6.2
    CPU: (10) arm64 Apple M1 Pro
    Memory: 228.30 MB / 32.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 21.6.1 - /opt/homebrew/bin/node
    Yarn: 1.22.19 - ~/.yarn/bin/yarn
    npm: 10.2.4 - /opt/homebrew/bin/npm
    pnpm: 8.15.2 - ~/Library/pnpm/pnpm
  Browsers:
    Chrome: 123.0.6312.59
    Safari: 16.6
  npmPackages:
    @sveltejs/kit: ^2.0.0 => 2.4.3 
    bits-ui: ^0.15.1 => 0.15.1 
    formsnap: ^0.5.0 => 0.5.0 
    lucide-svelte: ^0.314.0 => 0.314.0 
    mode-watcher: ^0.2.2 => 0.2.2 
    shadcn-svelte: ^0.6.1 => 0.6.1 
    svelte: ^4.2.7 => 4.2.9 
    svelte-sonner: ^0.3.19 => 0.3.19 
    sveltekit-superforms: ^2.5.0 => 2.5.0
```

### Severity

blocking an upgrade

**Comment by huntabyte** [comment]
Not able to reproduce. It's importing from the correct path:
`$lib/components/ui/toggle/index.js`

You use the extension it gets compiled to, not the extension in its precompiled state. (see [this video](https://www.youtube.com/watch?v=8ORIzvgNWhU) for more info)

**Comment by coryvirok** [comment]
Hrmm, but the import from that registry path was what showed up in my code after the shadcn cli installed toggle-group. Here's a repro - https://stackblitz.com/edit/github-6lpr3i-rgedcr?file=src%2Flib%2Fcomponents%2Fui%2Ftoggle-group%2Findex.ts

**Comment by AdrianGonz97** [comment]
It looks like you have the CLI installed locally, which is not something I would recommend. Additionally, the local version is fairly out of date.

Instead, you can remove the local package and execute it with `npx shadcn-svelte@latest`.

**Comment by coryvirok** [comment]
That did the trick. Thanks!

**Comment by coryvirok** [comment]
Maybe I'm going crazy, but it looks like this is actually broken now. I just tried using a fresh sveltekit project and v0.10.0 of the CLI. I'm installing using `npx shadcn-svelte@latest add toggle-group` and I've verified there is no local shadcn-svelte package installed.

Repro here: https://stackblitz.com/edit/stackblitz-starters-7qrlxl?file=my-app%2Fsrc%2Flib%2Fcomponents%2Fui%2Ftoggle-group%2Ftoggle-group-item.svelte

**Comment by AdrianGonz97** [comment]
@coryvirok How do you mean by broken? Are you having the same issue as before? Or are you getting a different error? I just created a new SK project and everything is working with `v0.10.0`. The provided repro is also working for me as well.

**Comment by coryvirok** [comment]
The issue im seeing is that the code that ends up in toggle-group.svelte appears to be importing a ".js" file. This breaks my sveltekit dev server since that file doesn't exist during development.

Sorry for the mobile screenshot, but here's what I'm seeing.

![Screenshot_20240329-174304_Chrome](https://github.com/huntabyte/shadcn-svelte/assets/769323/b58467fc-2553-4b46-92da-627400b07541)

**Comment by AdrianGonz97** [comment]
> The issue im seeing is that the code that ends up in toggle-group.svelte appears to be importing a ".js" file (note I'm using typescript). This breaks my sveltekit dev server since that file doesn't exist during development.

Import paths using the `.js` extension are normal and expected, even when using TS (as @huntabyte pointed out at the top of this thread). This is just how TS works. [Import paths are left untouched during compilation](https://www.typescriptlang.org/docs/handbook/modules/theory.html#module-specifiers-are-not-transformed) and bundlers know how to resolve these paths properly as well. We use it [everywhere](https://github.com/huntabyte/shadcn-svelte/blob/main/apps/www/src/routes/(app)/%2Bpage.svelte) without issue.

> This breaks my sveltekit dev server since that file doesn't exist during development.

I'll need to see the specific error to get an understanding of the issue. I was able to get the linked stackblitz reproduction working without much issue. The only thing I noticed was that I had to `cd my-app` before running any scripts since the SK project doesn't reside in the root. Outside of that, everything's working as expected.
